src/assets/js/search.js

```diff
@@ -0,0 +1,74 @@
+(() => {
+  const getExcerpt = (content, query) => {
+    const queryRegex = new RegExp(query, "i");
+    const searchIndex = content.search(queryRegex);
+    const queryLength = query.length;
+    const excerpt = content.slice(
+      searchIndex - 100,
+      searchIndex + queryLength + 100
+    );
+
+    return excerpt.replace(
+      queryRegex,
+      "<strong class='search-keyword'>$&</strong>"
+    );
+  };
+
+  const displaySearchResults = (results, store, searchTerm) => {
+    const searchResults = document.getElementById("search-results");
+
+    if (results.length) {
+      let appendString = "";
+
+      for (let i = 0; i < results.length; i++) {
```

Comment:
Are you using this syntax over `.forEach` for performance?

Reply:
Nope, good point!

Reply:
I stopped looking at the details of the syntax in that file - I think some of the stuff that came from the guide can probably be refactored/cleaned up a little, but it can probably come later!